design/API.yml

```diff
@@ -735,13 +566,10 @@ paths:
                   - type: object
                     properties:
                       data:
-                        type: object
```

Comment:
+2
...
Maybe you have not merge dev and resolve conflicts correctly. Do it now.